CPAC/pipeline/cpac_group_runner.py

```diff
@@ -1746,26 +1760,26 @@ def run_isc_group(pipeline_dir, out_dir, working_dir, crash_dir,
 
     output_df_dct = gather_outputs(
         pipeline_dir,
-        ["functional_to_standard", "roi_timeseries"],
+        ["space-template_bold", "desc-Mean_timeseries"],
         inclusion_list=None,
         get_motion=False, get_raw_score=False, get_func=True,
-        derivatives=["functional_to_standard", "roi_timeseries"],
-        exts=['nii', 'nii.gz', 'csv']
+        derivatives=["space-template_bold", "desc-Mean_timeseries"],
+        #exts=['nii', 'nii.gz', 'csv']
     )
 
     iteration_ids = []
-    for preproc_strat in output_df_dct.keys():
+    for preproc_strat in output_df_dct.items():
         # go over each preprocessing strategy
 
         derivative, _ = preproc_strat
```

Comment:
I assume this is working, but I wonder why this change would be necessary. Like https://github.com/FCP-INDI/C-PAC/blob/6dc4a2b904529504a21a91252d4dbfa3b4418fc8/CPAC/pipeline/cpac_group_runner.py#L1794 uses `preproc_strat` as a key. Is that key a tuple like you'd get out of `output_df_dct.items()`?

Reply:
I think so, its from the original code and didn't appear to be an error when I ran it.

Reply:
I see, there are 2 other instances where this appears and it seems to do the same thing. However, I think .keys() is correct instead of .items().

Reply:
Resolved in https://github.com/FCP-INDI/C-PAC/pull/1787/commits/92f9658a869432f86e7e2cbfcfb8f4a9a8675f9c